malaria24/settings/dev.py

```diff
@@ -2,8 +2,6 @@
 
 
 DEBUG = True
-TEMPLATE_DEBUG = True
```

Comment:
you can leave these settings in. There are still templates for the admin and django-rest-framework